### ISSUE TITLE
New lines in templates

### ISSUE THREAD

**Issue by jtdLab** [felangel/mason#683]
Im having trouble with new lines and empty lines when writing templates for mason.

For example:

Given `a` is `true` and `b` is `true`

Desired Output:
```
A
B
```

Given `a` is `false` and `b` is `true`

Desired Output:
```
B
```

Given `a` is `true` and `b` is `false`

Desired Output:
```
A
```

Given `a` is `true` and `b` is `true`

Desired Output:
```
A
B
```

How would the template of this scenario look like?

**Comment by spydon** [comment]
@wolfenrain gave me a work-around:
```yaml
packages:{{#containsApps}}
  - apps/*{{/containsApps}}{{#containsPackages}}
  - packages/*{{/containsPackages}}
```

It has very bad readability though. :sweat_smile:

**Comment by spydon** [comment]
We were talking about whether this is according to the mustache spec or not too, and it is not:
https://github.com/mustache/spec/blob/master/specs/sections.yml#L279-L291

I guess this issue really belongs on the parsing library, but since that one doesn't have an issue section and isn't maintained I'll put the info here anyways, in the case that the parsing library will be changed in the future.

**Comment by felangel** [comment]
> We were talking about whether this is according to the mustache spec or not too, and it is not: https://github.com/mustache/spec/blob/master/specs/sections.yml#L279-L291
> 
> I guess this issue really belongs in the parsing library, but since that one doesn't have an issue section and isn't maintained I'll put the info here anyways, in the case that the parsing library will be changed in the future.

Good call. I’ll see if I can fix this in the underlying mustache library, thanks!
